repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/AccessTypeRolesEntity.java

```diff
@@ -0,0 +1,203 @@
+package uk.gov.hmcts.ccd.definition.store.repository.entity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import static javax.persistence.FetchType.LAZY;
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Table(name = "access_type_roles")
+@Entity
+public class AccessTypeRolesEntity implements Serializable {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = IDENTITY)
+    private Integer id;
+
+    @Column(name = "live_from")
+    private LocalDateTime liveFrom;
+
+    @Column(name = "live_to")
+    private LocalDateTime liveTo;
+
+    @ManyToOne(fetch = LAZY)
+    @JoinColumn(name = "case_type_id", nullable = false)
+    private CaseTypeEntity caseTypeId;
+
+    @Column(name = "access_type_id")
```

Comment:
I think we'd probably need to add nullable = false to some more of these fields - see required on JIRA